doc/source/man/bandit.rst

```diff
@@ -25,51 +25,80 @@ OPTIONS
 =======
 
   -h, --help            show this help message and exit
+
```

Comment:
Can you please undo the line spacing. The man page options here are meant to look similar to running `bandit --help`

Reply:
Sure, I was using a rst validator to try and get the github preview to parse correctly